src/app/Controllers/ActorsController.cs

```diff
@@ -28,49 +31,39 @@ public ActorsController(ILogger<ActorsController> logger, IDAL dal)
         }
 
         /// <summary>
-        /// Returns a JSON array of Actor objects
+        /// Returns a JSON array of Actor objects based on query parameters
         /// </summary>
-        /// <param name="q">(optional) The term used to search Actor name</param>
-        /// <param name="pageNumber">1 based page index</param>
-        /// <param name="pageSize">page size (1000 max)</param>
-        /// <response code="200">JSON array of Actor objects or empty array if not found</response>
+        /// <param name="actorQueryParameters"></param>
         [HttpGet]
-        public async Task<IActionResult> GetActorsAsync([FromQuery] string q, [FromQuery] int pageNumber = 1, [FromQuery] int pageSize = Constants.DefaultPageSize)
+        public async Task<IActionResult> GetActorsAsync([FromQuery]ActorQueryParameters actorQueryParameters)
```

Comment:
space between [FromQuery] and ActorQueryParameters

Reply:
i see a couple more spots across files with this minor lint issue

Reply:
fixed in new commit.